### PR TITLE
Added the option to run the auxiliary process ('-A')

### PR DESCRIPTION
**Pull request by raylu-hdf**
For now, it's enabled by hand only for testing.

### CODE REVIEW

File: test/vfd_swmr_bigset_writer.c [hunk 1]
```diff
@@ -1148,6 +1154,101 @@ notify_reader(np_state_t *np, unsigned step)
     return -1;
 }
 
+/*-------------------------------------------------------------------------
+ * Function:    md_ck_cb()
+ *
+ * Purpose:     This is the callback function for debugging only.  It's used
+ *              when the H5F_ACS_GENERATE_MD_CK_CB_NAME property is set in fapl.
+ *                  --Opens and read the metadata file into a buffer.
```
Comment (kyang2014) [inline test/vfd_swmr_bigset_writer.c:1162]: minor typo: "Open and read" instead of "Opens and read"

**Review by kyang2014 — APPROVED** [review]
I haven't attended the meetings recently and don't know the status of this work. Just from the code review, the changed code looks fine to me.

**Comment by raylu-hdf** [comment]
> I haven't attended the meetings recently and don't know the status of this work. Just from the code review, the changed code looks fine to me.

I'm going to update this PR with some significant changes this week.  If possible, please review it again after I update it.

**Comment by raylu-hdf** [comment]
Instead of updating this one, I'll make another PR because of too many changes.

**Pull request closed without merging**